core/platform/image/gae_image_services_test.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for methods in the gae_image_services."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+
+from constants import constants
+from core.platform.image import gae_image_services
+from core.tests import test_utils
+import feconf
+import python_utils
+
+
+class GaeImageServicesUnitTests(test_utils.GenericTestBase):
```

Comment:
There are other tests you could write. E.g. equality tests of the compression results against "golden files" that are also stored in core/tests/data, a check that the format of the resulting file is correct, etc.

Reply:
Done